### PR TITLE
refactor cookie-policy page

### PR DESCRIPTION
**Pull request by ggsbv**
* move tables to page center
* refine positioning
* update careers-banner title positioning
* center careers-banner title when there's no subtitle

### CODE REVIEW

File: version_control/Codurance_September2020/css/pages/cookie-policy.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
+{% import '../utils/utils.css' as utils %}
+
+.body-container-cookie-policy h2 {
+    font-size: var(--dudler-font-size);
+    line-height: var(--dudler-line-height);
```
Comment (Dan-Bird) [inline version_control/Codurance_September2020/css/pages/cookie-policy.css:5]:
```suggestion
    {{ utils.dudler() }}
```

File: version_control/Codurance_September2020/css/pages/cookie-policy.css [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    font-weight: var(--heavy-font-weight);
+    margin-bottom: var(--dudler-margin);
+    text-align: center;
+}
+
+.body-container-cookie-policy h3 {
+    font-size: var(--base-font-size);
+    line-height: var(--base-line-height);
```
Comment (Dan-Bird) [inline version_control/Codurance_September2020/css/pages/cookie-policy.css:13]:
```suggestion
    {{ utils.base() }}
```

File: version_control/Codurance_September2020/css/pages/cookie-policy.css [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+.custom-cookie-policy-cookie-details-section h3 {
+    text-align: center;
+}
+
+.custom-cookie-policy-cookie-details-section th, .custom-cookie-policy-cookie-details-section td {
+    vertical-align: middle;
+}
+
+.custom-cookie-policy-cookie-details-section table {
+    text-align: center!important;
```
Comment (Dan-Bird) [inline version_control/Codurance_September2020/css/pages/cookie-policy.css:59]: Why does this need `!important` ?

File: version_control/Codurance_September2020/css/pages/cookie-policy.css [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+    margin: 0 auto;
+    padding: 0;
+}
+
+.custom-cookie-policy-cookie-details-section table thead tr {
+    border-bottom: 1px solid #444;
+}
+
+.custom-cookie-policy-cookie-details-section table thead tr th {
+    {{ utils.natus() }}
```
Comment (Dan-Bird) [inline version_control/Codurance_September2020/css/pages/cookie-policy.css:71]:
```suggestion
    font-size: var(--natus-font-size);
    line-height: var(--natus-line-height);
```

File: version_control/Codurance_September2020/css/pages/cookie-policy.css [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+}
+
+.custom-cookie-policy-cookie-details-section h4:after {
+    content: "";
+    width: 60px;
+    display: block;
+    height: 1px;
+    background: var(--persimmon);
+    margin-top: 16px;
+}
```
Comment (Dan-Bird) [inline version_control/Codurance_September2020/css/pages/cookie-policy.css:112]: Are there any h4s on the page?

**Review by Dan-Bird — COMMENTED** [review]
Just a couple of minor things raised.

File: version_control/Codurance_September2020/css/pages/cookie-policy.css (hunk 5 above)
Comment (ggsbv) [reply version_control/Codurance_September2020/css/pages/cookie-policy.css:112]: nice catch, I've removed them

File: version_control/Codurance_September2020/css/pages/cookie-policy.css (hunk 3 above)
Comment (ggsbv) [reply version_control/Codurance_September2020/css/pages/cookie-policy.css]: it doesn't. burn, baby, burn🔥

**Pull request merged**